scripts/start.js

```diff
@@ -42,11 +60,22 @@ switch(argv.mode) {
     concurrently(
       [
         startSources,
-        startCore,
+        startCoreProd,
```

Comment:
`startCoreProd` -> omdat alleen in `--prod` de react bindings worden gebakken?

Reply:
Ja